app/models/review.rb

```diff
@@ -4,5 +4,7 @@ class Review < ApplicationRecord
   belongs_to :user
   belongs_to :space
 
-  validates :star_rating, presence: true
+  validates :star_rating, presence: true, numericality: { greater_than: 0, less_than: 6 }
+
+  after_save { space.aggregate_star_rating }
```

Comment:
Should also run on `after_destroy` to make sure the aggregation is handled when a review is removed.

Reply:
Yep good point :)